services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,27 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to reset meters for unmetered flow"() {
+        given: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields"
+        def availableSwitches = topology.activeSwitches
+        def src = availableSwitches[0]
+        def dst = availableSwitches[1]
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
```

Comment:
Not sure, that we need this 'assume'.

Reply:
Agree. It kind of makes sense, but we by default assume that our topology has 2+ switches available. Otherwise we should have put this assumption in 80% of tests

Reply:
Done.